packages/rancher-logging/overlay/templates/loggings/rke2/daemonset.yaml

```diff
@@ -31,11 +31,69 @@ spec:
           nodeSelector:
             {{- toYaml . | nindent 6 }}
           {{- end }}
+      serviceAccountName: "{{ .Release.Name }}-rke2-journald-aggregator"        
       volumes:
         - name: logdir
           hostPath:
             path: /etc/rancher/logging/logs/
         - name: config
           configMap:
             name: "{{ .Release.Name }}-rke2"
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: "{{ .Release.Name }}-rke2-journald-aggregator"
+  namespace: "{{ .Release.Namespace }}"
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: "{{ .Release.Name }}-rke2-journald-aggregator"
+rules:
+  - apiGroups:
+    - policy
+    resourceNames:
+    - "{{ .Release.Name }}-rke2-journald-aggregator"
+    resources:
+    - podsecuritypolicies
+    verbs:
+    - use
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: "{{ .Release.Name }}-rke2-journald-aggregator"
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: "{{ .Release.Name }}-rke2-journald-aggregator"
+subjects:
+  - kind: ServiceAccount
+    name: "{{ .Release.Name }}-rke2-journald-aggregator"
+---
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: "{{ .Release.Name }}-rke2-journald-aggregator"
+  namespace: "{{ .Release.Namespace }}"
+spec:
+  allowPrivilegeEscalation: false
+  allowedHostPaths:
+  - pathPrefix: /etc/rancher/logging/logs
```

Comment:
Is this really all thats read ? cc @nickgerace

Reply:
At least, it's the only volume I could find in the Daemonset: https://github.com/rancher/charts/pull/931/files/e3c75a2e2930663762440b044d83fb965f4729bb#diff-725e07f25e638efd8f1864b278104ab6d92602490dd07e0ebaf3fe4d6381fd3fR35-R38

Reply:
As far as RKE2 goes, I believe that is correct @cbron. (cc @paynejacob optionally if you have more context)

Reply:
Yes, this is the link between our fluenbit pods and banzai's pods.  Their configuration is too restrictive to collect all the rke2 logs so we aggregate them and present them in a way banzai can consume in this file.